test/extended/util/framework.go

```diff
@@ -326,18 +323,32 @@ func WaitForOpenShiftNamespaceImageStreams(oc *CLI) error {
 	// we've also determined that e2e-aws-image-ecosystem can be started before all the operators have completed; while
 	// that is getting sorted out, the longer time will help there as well
 	e2e.Logf("Scanning openshift ImageStreams \n")
-	success := false
-	wait.Poll(10*time.Second, 150*time.Second, func() (bool, error) {
-		success = scan()
-		return success, nil
+	var scanErr error
+	pollErr := wait.Poll(10*time.Second, 150*time.Second, func() (bool, error) {
+		scanErr = scan()
+		if scanErr != nil {
+			return false, nil
+		}
+		return true, nil
 	})
-	if success {
+	if pollErr == nil {
 		e2e.Logf("Success! \n")
 		return nil
 	}
 	DumpImageStreams(oc)
 	DumpSampleOperator(oc)
-	return fmt.Errorf("Failed to import expected imagestreams")
+	errorString := ""
+	if strings.Contains(errorString, "FailedImageImports") {
```

Comment:
doesn't look like you're setting errorString correctly here

Reply:
ahhh ... classic last minute change caught me ... thanks